listenbrainz/webserver/__init__.py

```diff
@@ -147,9 +148,14 @@ def create_app(debug=None):
     # Static files
     import listenbrainz.webserver.static_manager
     static_manager.read_manifest()
-    app.context_processor(lambda: dict(get_static_path=static_manager.get_static_path))
     app.static_folder = '/static'
 
+    from listenbrainz.webserver.utils import inject_global_props
```

Comment:
any reason why this item (and the static manager) are imported in this method and not at the top of the file? I could believe that it's because of circular imports, but if that's not the case then let's put them at the top